AdminUi/test/AdminUi.Tests.Integration/Models/HttpResponse.cs

```diff
@@ -10,3 +10,11 @@ public class HttpResponse<T>
     public string? ContentType { get; set; }
     public string? RawContent { get; set; }
 }
+
+public class HttpResponse
+{
+    public HttpStatusCode StatusCode { get; set; }
+    public bool IsSuccessStatusCode { get; set; }
+    public string? RawContent { get; set; }
+    public Error? Error { get; set; }
+}
```

Comment:
Why didn't you add the `ContentType` property?

Reply:
Don't really see a use for it. It what instance would it be relevant?

Reply:
Oh, sure. We don't have a content, so ContentType doesn't make sense.